docs/DeveloperGuide.adoc

```diff
@@ -295,35 +297,127 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|`* * *` |organized user |add items to shopping list |decide the grocery items that I need to purchase later
 
-|`* * *` |user |add a new person |
+|`* * *` |organized user |mark grocery items as bought and add their corresponding expiry and amount details|keep track of bought groceries
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |lazy user |add all bought items to grocery list |update grocery to contain all the bought items
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |meticulous user |display shopping list |view the items in shopping list (items yet to be bought before bought items)
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+|`* *` |organized user |tag items (not yet bought) in shopping list as urgent |view the urgent items first when I display shopping list
+
+|`* * *` |careless user |edit and delete items in shopping list |change details of or remove grocery items
+
+|`* * *` |user |add <<template,templates>> of grocery items |to keep track of what I want to have in my fridge
+
+|`* * *` |user |edit templates of grocery items |edit items and volumes based on my current needs and preferences
+
+|`* * *` |user |delete templates |maintain a short list of templates I regularly use
+
+|`* * *` |user |add multiple templates |use different templates depending on the occasion
+
+|`* * *` |user |generate a shopping list using my templates |save time on checking what is missing from my fridge
+
+|`* * *` |environmentally-conscious user |compare my <<food-waste,food wastage statistics>> |better manage my food waste
+
+|`* * *` |environmentally-conscious user |get feedback on how I am performing on my food waste management |improve my food waste management
+
+|`* *` |environmentally-conscious user |see which kinds of food I most commonly waste |cut down on unnecessary food waste
```

Comment:
Can consider listing from most important requirements before moving to less important requirements